src/main/java/bio/terra/common/exception/BadRequestException.java

```diff
@@ -0,0 +1,34 @@
+package bio.terra.common.exception;
+
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+/**
+ * This base class has data that corresponds to the ErrorReport model generated from the OpenAPI
+ * yaml. The global exception handler auto-magically converts exceptions of this base class into the
+ * appropriate ErrorReport REST response.
+ */
+public class BadRequestException extends ErrorReportException {
```

Comment:
this javadoc comment doesn't seem to match this class. Ditto others. Let's have the same kind of comment for all the sub classes of ErrorReportException.